README.md

```diff
@@ -67,16 +67,29 @@ Spotlight introduces functionality that depends on being able to send emails to
 
 See the [Spotlight wiki](https://github.com/sul-dlss/spotlight/wiki) for more detailed information on configuring Spotlight.
 
-## To start the development/test application
+# Developing Spotlight
 
- 1. Clone this repo, `cd` in and run `$ bundle install`
- 2. Start jetty and the dev server: `$ bundle exec rake spotlight:server` (this task will build a Spotlight-based application, start Solr, and run the built-in rails server)
- 3. Visit [http://localhost:3000](http://localhost:3000)
+Spotlight:
+
+* is a Rails engine and needs to be used in the context of a Rails application. We use [engine_cart](https://github.com/cbeer/engine_cart) to create an internal test application at spec/internal.
+* uses Solr as part of its integration tests. We use [jettywrapper](https://github.com/projecthydra/jettywrapper) to manage the Solr instance used for development and test.
+
+Our `$ rake ci` and `$ rake spotlight:server` tasks utilize Solr and the testing rails app automatically.
+
+##  More Information for Developers
+
+* [Contributing to Spotlight](https://github.com/sul-dlss/spotlight/wiki/Contributing-to-Spotlight)
+* [Testing](https://github.com/sul-dlss/spotlight/wiki/Testing)
 
 ## Tests
 
-Run tests:
+### Prerequisites
+
+PhantomJS (https://github.com/teampoltergeist/poltergeist#installing-phantomjs) is an addition requirement for testing javascript.
+
+### Run all the tests:
 
 ```
-$ rake
+$ rake ci
```

Comment:
My opinion FWIW, I like knowing what is happening under the hood as a developer.  I like the last suggestion of breaking out more detail about how you can run things separately. I don't like it when it says little, assumes I know about the convention of "rake" runs the tests (maybe I am newish Ruby developer and don't), and then on top of it also runs style conventions, code coverage and other stuff that is not obvious.  In a way, it kind of reminds me of Rails way back in the earlier days when you could do scaffolding and magic would happen.  It was cool, but bugged me that I didn't know what was going on.

So I guess I like the idea of a TL/DR at the top which basically lists each command you need to get things running, including tests with "rake".  And then a bit of an extended section explaining how to do things like reload fixtures; clear the state of the jetty, run tests independently of things like rubucop, what "rake" actually does, etc.